tests/python_client/requirements.txt

```diff
@@ -29,7 +29,7 @@ python-benedict==0.24.3
 timeout-decorator==0.5.0
 
 # version need to be consistent with protobuf used in pymilvus
-protobuf==3.17.1
+protobuf==4.21.5
```

Comment:
why we update the protobuf? did pymilvus update the same version accordingly? If not, update protobuf will decrease the search and insert performance :(